adapters/Vungle/VungleAdapter/Bidding/GADMediationVungleNativeAd.h

```diff
@@ -20,8 +20,12 @@
 /// Asks the receiver to render the ad configuration.
 - (nonnull instancetype)initNativeAdForAdConfiguration:(nonnull GADMediationNativeAdConfiguration *)adConfiguration
                                      completionHandler:(nonnull GADMediationNativeLoadCompletionHandler)completionHandler;
+
+/// Constructor is unavailable. Please use initWithAdConfiguration:completionHandler:.
 - (nonnull instancetype)init NS_UNAVAILABLE;
-- (void)requestAd;
+
+/// Requests a rewarded ad from Vungle.
```

Comment:
A little confusing. Do we request a native ad or a rewarded ad from Vungle?  I saw the class name is `VungleNativeAd`. Thanks a lot. :)

Reply:
Yes. I copied the comment and missing changing this one. Thanks for catching.